src/config/jwt_compatible_hash.cr

```diff
@@ -0,0 +1 @@
+alias JWTCompatibleHash = Hash(String, String | Int32 | Bool?)
```

Comment:
A file with a single alias? Why it isn't put in an already existing module, like `DppmRestApi`?

Reply:
I was trying to think of a place to put it where it fit with other stuff, and there just didn't seem to be any. It doesn't seem like a good idea to clutter the main file with stuff like this, and putting it anywhere else would make it hard to find. It doesn't do any harm being there. We could put it back in `src/utils.cr` but you seemed very much against the idea of having that at all.

Reply:
You need it at `src/config/user.cr`, so it can be defined there.

Reply:
Ok, but if we find a reason to use that type outside of User then we should move it from there, but for now that looks fine.